test/build.js

```diff
@@ -139,6 +139,17 @@ describe('grunt', function() {
         done();
       });
     });
+
+    // Ensure the build/html/config directory exists if it should.
+    it('config directory should exist if provided ', function(done) {
+      var src = fs.existsSync('src/config');
+      if (src) {
+        fs.existsSync('build/html/config', function(exists) {
+          assert.ok(exists);
```

Comment:
Maybe I'm fuzzy on this, but if we have a test where the src/config doesn't exist, we should confirm the build/html/config doesn't exist. If we don't, why include the check?

Reply:
✅